Gems/WhiteBox/Code/Source/Viewport/WhiteBoxViewportConstants.h

```diff
@@ -29,6 +29,14 @@ namespace WhiteBox
     AZ_CVAR_EXTERNED(float, cl_whiteBoxEdgeVisualWidth);
     AZ_CVAR_EXTERNED(float, cl_whiteBoxEdgeSelectionWidth);
     AZ_CVAR_EXTERNED(float, cl_whiteBoxSelectedEdgeVisualWidth);
+    
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxVertexHover);
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxPolygonHover);
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxOutlineHover);
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxPolygonSelection);
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxVertexSelection);
+    AZ_CVAR_EXTERNED(AZ::Color, ed_whiteBoxOutlineSelection);
+    AZ_CVAR_EXTERNED(float, ed_whiteBoxPolygonViewOverlapOffset);
```

Comment:
update: See my earlier comment about the other existing draw functions - we might want to try unify them all so we don't get drifting functionality and have a consistent function to use. If you could take a little look at this that'd be great. Thanks!

Reply:
the plan was to describe the action and just update everything to follow that pattern. avoid having to define colors for everything. would it be better to define it for everything or for the use case :?

Reply:
here is a PR that I prepared for this: https://github.com/o3de/o3de/pull/10510